scripts/ci/image_build.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+GITLAB_BRANCH=master
+# URL to trigger the build of the Docker image
+# Set the project id: you can find your trigger URL under "Pipeline Triggers" at Settings/CI-CD
+# https://gitlab.cern.ch/<your-group>/<your-repo>/settings/ci_cd
+pipeline() {
+    echo $trigger_uri
+    GITLAB_PIPELINE_TRIGGER_URL=$trigger_uri
+    GITLAB_PIPELINE_TRIGGER_TOKEN=$token
+    if [ "$1" = "ui" ]
+    then
+        APPLICATION_IMAGE_NAME='ui'
+    elif [ "$1" = "analysispreservationimage" ]
+    then
+        APPLICATION_IMAGE_NAME='analysispreservationimage'
+    fi
+}
+
+usage() { echo "Usage: $0 [-b <branch> | -c <commit> | -t <tag> | -p <pull-request>] [-d <deploy-image>]" 1>&2; exit 1; }
+
+# Source code inside the Docker image will be cloned on a specific `branch` (default `master`), `commit`, `tag` or `pr`
+git_ref_params=0
+while getopts b:t:p:d:i opt
```

Comment:
- [ ] lets add back `d - deploy` option feature and respective code
- [ ] change `d` option (since it is used in the master script )  to `i` for image